testCases/casual/001.single.service.scale.payload.py

```diff
@@ -36,31 +33,31 @@ class TestCase( FastHttpUser):
     def task1( self):
         self.client.post(
             name = "1B",
-            url = "/casual/example/echo",
+            path = "/casual/example/echo",
```

Comment:
We need to decide which version of locust we shall support in this path/url turmoil.

Reply:
why not choose neither? if whe keep it as the first parameter we can omit the name, and once we've settled on a version we can reintroduce the named parameter if we want to

Reply:
removed path/url. tested ok with locust 2.8.6 and 2.24.1